EnclaveWrapper/EnclaveWrapper/SgxException.cpp

```diff
@@ -0,0 +1,19 @@
+#include "stdafx.h"
+#include "SgxException.h"
+
+#define BUFSIZE 256
+
+SgxException::SgxException(sgx_status_t errorCode) {
+	this->errorCode = errorCode;
+	buf = new char[BUFSIZE];
+}
+
+SgxException::~SgxException() {
+	delete buf;
```

Comment:
This way you can free only first element under the pointer. To free whole array you should use: `delete[] buf;`